### PR TITLE
Allow the Skim tasl to select events with a track within a given pt bin

### PR DESCRIPTION
**Pull request by fkrizek**

### CODE REVIEW

File: PWG/EMCAL/EMCALtasks/AliAodSkimTask.cxx [hunk 1]
```diff
@@ -157,7 +157,7 @@ void AliAodSkimTask::UserExec(Option_t *)
 
   // Accept event only if an track with a minimum pT of fTrackMinPt has been found in the event
   Bool_t storePt = kFALSE;
-  if (fTrackMinPt > 0 ){
+  if (fTrackMinPt > 0 && fTrackMaxPt < 0){
```
Comment (loizides) [inline PWG/EMCAL/EMCALtasks/AliAodSkimTask.cxx]: Not sure about this upper check, should we not check the "or" if any of the two are turned on (-1 means not used)

Comment (fkrizek) [reply PWG/EMCAL/EMCALtasks/AliAodSkimTask.cxx]: In principle the default situation is fTrackMaxPt=-1    so  if fTrackMaxPt is not set by user  and you give just fTrackMinPt > 0   the macro should behave as before,  you select all events with pT> fTrackMinPt .

File: PWG/EMCAL/EMCALtasks/AliAodSkimTask.cxx [hunk 2]
```diff
@@ -171,6 +171,23 @@ void AliAodSkimTask::UserExec(Option_t *)
     storePt = kTRUE;
   }
 
+  // Accept event only if an track within a pT interval (fTrackMinPt, fTrackMaxPt) has been found in the event
```
Comment (loizides) [inline PWG/EMCAL/EMCALtasks/AliAodSkimTask.cxx]: I would have preferred a solution where we have one loop that decides all cases depending on the value of fTrackMinPt and fTrackMaxPt (ie only check the dependence if they are different from -1)

Comment (fkrizek) [reply PWG/EMCAL/EMCALtasks/AliAodSkimTask.cxx]: OK i will change the code

**Pull request closed without merging**